x/archivedb/db_test.go

```diff
@@ -0,0 +1,170 @@
+// Copyright (C) 2019-2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package archivedb
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/ava-labs/avalanchego/database"
+	"github.com/ava-labs/avalanchego/database/memdb"
+)
+
+func getBasicDB() (*archiveDB, error) {
+	return NewArchiveDB(
+		context.Background(),
+		memdb.New(),
+	)
+}
+
+func TestDbEntries(t *testing.T) {
```

Comment:
can we add a test that calls `Put` After `Delete` for the same key on the batch and also separately a test that calls `Delete` after `Put`?

Reply:
That is being covered by `TestInterface`